test/visual-regression/util/get-file-names.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
+const mkdirp = require('mkdirp');
+const { baseUrl } = require('../../e2e/e2e-helpers');
+
+const screenshotDirectory = path.join(__dirname, '../../../logs/visual-regression');
+const baselineDir = path.join(screenshotDirectory, '/baseline');
+const diffDir = path.join(screenshotDirectory, '/diffs');
+
+// Takes a full URL and translates that into a filename in the logs directory to store images.
+function getFileNames(route) {
+  const uri = route.replace(baseUrl, '');
+  const baseline = path.join(baselineDir, `/${uri}.png`);
+  const diff = path.join(diffDir, `/${uri}.png`);
+
+  return [baseline, diff];
+}
+
+// Creates a directory if it doesn't exist already.
+// Necessary when writing out the diff image.
+function createDirectoryIfNotExist(filePath) {
+  const directory = path.dirname(filePath);
+  return new Promise((resolve, reject) => {
+    mkdirp(directory, err => {
+      return err ? reject(err) : resolve();
```

Comment:
I saw how many downloads `mkdir` had and that was it. Actually, I was thinking of removing the dependency and just right it ourselves real quick with `.reduce`. No strong feelings at all in that decision really.

Reply:
Heh, me neither. :man_shrugging: I like not adding dependencies, but I guess it depends how complicated it'd be without it.

Reply:
I'll wait on this one to see what others think before I change on it, because it would be a sort of ugly function to implement that ourselves I think.